qp_shotgun/kneaddata/kneaddata.py

```diff
@@ -273,6 +284,6 @@ def kneaddata(qclient, job_id, parameters, out_dir):
     # Step 4 generating artifacts
```

Comment:
Do we need to add a gzip compression step after kneaddata so that we're not keeping unzipped files around on the system?